api-connector/src/main/java/com/adnovum/vcms/connector/config/SwaggerConfig.java

```diff
@@ -0,0 +1,25 @@
+package com.adnovum.vcms.connector.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import springfox.documentation.builders.ApiInfoBuilder;
+import springfox.documentation.builders.PathSelectors;
+import springfox.documentation.builders.RequestHandlerSelectors;
+import springfox.documentation.spi.DocumentationType;
+import springfox.documentation.spring.web.plugins.Docket;
+
+@Configuration
+public class SwaggerConfig {
+
+	@Bean
+	public Docket api() {
+		return new Docket(DocumentationType.OAS_30).select()
+				.apis(RequestHandlerSelectors.basePackage("com.adnovum.vcms.connector.controller")).paths(PathSelectors.any())
+				.build().apiInfo(
+						new ApiInfoBuilder().title("VCMS Connector API").description("Create and track DIDComm " + "connections")
+								.version("1.2.1").build());
```

Comment:
not sure you want to spend time on it, but version number could come from outside

Reply:
Good point, it is copy paste heavy which is not nice. I'll double check how to make this more dynamic. :/